README.md

```diff
@@ -176,3 +179,47 @@ hocr-split file.html pattern
 
 Split a multipage hOCR file into hOCR files containing one page each.
 The pattern should something like "base-%03d.html"
+
+## Unit tests
+
+The unit tests are written using the [tsht](https://github.com/kba/tsht) framework.
+
+### Running the full test suite:
+
+```sh
+./test/tsht
+```
+
+### Running a single test
+
+```sh
+./test/tsht <path-to/unit-test.tsht>
+```
+
+e.g.
+
+```sh
+./test/tsht hocr-pdf/test-hocr-pdf.tsht
+```
+
+### Writing a test
+
+Please see the documentation in the [tsht](https://github.com/kba/tsht) repository and
+take a look at the existing [unit tests](./test/).
+
+1) Create a new directory under `./test`
+2) Copy any test assets (images, hocr files...) to this directory
```

Comment:
No, they can be shared, it's just easier to keep tests self-contained when code and assets are in the same place. Otherwise you run the risk of having some content that is not actually used for tests.